### ISSUE TITLE
multiple assignment in tests confuses beginners

### ISSUE THREAD

**Issue by ashleygwilliams** [rmurphey/js-assessment#100]
particularly in the functions section, speak step, this code was very confusing:
```
var speak = function() {
            return sayIt(this.greeting, this.name, '!!!');
          },
          obj = {
            greeting : 'Hello',
            name : 'Rebecca'
          };
```

simple changing it to this made it way easier to understand:
```
var speak = function() {
            return sayIt(this.greeting, this.name, '!!!');
          };
var obj = {
            greeting : 'Hello',
            name : 'Rebecca'
          };
```

**Comment by kadamwhite** [comment]
Strong +1. One `var` per assignment is much more obvious.

**Comment by jmeas** [comment]
:+1:

**Comment by ashleygwilliams** [comment]
:) i'll make the change. i love finding this stuff from really play testing the code during classes.

**Comment by ashleygwilliams** [comment]
solved by #101. closing.
